### DISCUSSION TITLE
can't split "endpoint" id into 2 parts when I check the trace related to slow SQL

### DISCUSSION THREAD

**Discussion by wangan0915** [apache/skywalking#10609] [Q&A] [answered]
oap version is 9.4.0 , web version id 9.4.0  java agent version is 8.6.0
When I check the trace related to slow SQL，i get  error about can't split "endpoint" id into 2 parts, Is my agent version too low?
![image](https://user-images.githubusercontent.com/39958329/228475381-911f44da-dca6-43be-b7cc-e3d87845b4b0.png)
![image](https://user-images.githubusercontent.com/39958329/228475458-9fdd2b1c-fbfd-484a-8e7e-a2c46fcd55da.png)

**Comment by wu-sheng** [comment, marked as answer]
Fixed through https://github.com/apache/skywalking-booster-ui/pull/247
